drv_cfg.go

```diff
@@ -170,7 +180,7 @@ func DrvCfgQuerySystems() (*[]ConfiguredCluster, error) {
 
 	output, err := executeFunc("chroot", "/noderoot", drvCfg, "--query_mdm")
 	if err != nil {
-		return nil, fmt.Errorf("DrvCfgQuerySystems: Request to query MDM failed : %v", err)
+		return nil, fmt.Errorf("drvCfgQuerySystems: Request to query MDM failed : %v", err)
```

Comment:
This is a good example: you cannot lowercase the first letter, since DrvCfgQuerySystems is a name. However, this name is redundant and should be removed altogether. The error message should provide information the caller doesn't yet know. Since the caller inherently knows the function name, it should be omitted. The message could be reduced to `fmt.Errorf("failed to query MDM: %v", err)`

Reply:
Fixed